entrypoint.sh

```diff
@@ -2,17 +2,27 @@
 
 set -Euo pipefail
 
+export VAULT_S3_TOKEN=$(cat /run/secrets/vault-s3-token)
+
 if [[ -f "initial_setup" ]]; then
-	if [[ -f "/run/secrets/cert.pem" ]]; then
-		cat /run/secrets/cert.pem >> /usr/local/lib/python3.7/site-packages/certifi/cacert.pem
-	fi
-	
-	ACCESS=$(cat /run/secrets/access)
-	sed -i s@\<MINIO_ACCESS_KEY\>@$ACCESS@ config.ini
-	
-	SECRET=$(cat /run/secrets/secret)
-	sed -i s@\<MINIO_SECRET_KEY\>@$SECRET@ config.ini
-	rm initial_setup
+    if [[ -f "/run/secrets/cert.pem" ]]; then
+        cat /run/secrets/cert.pem >> /usr/local/lib/python3.7/site-packages/certifi/cacert.pem
+    fi
+    
+    ACCESS=$(cat /run/secrets/access)
+    sed -i s@\<MINIO_ACCESS_KEY\>@$ACCESS@ config.ini
+    
+    SECRET=$(cat /run/secrets/secret)
+    sed -i s@\<MINIO_SECRET_KEY\>@$SECRET@ config.ini
```

Comment:
move the `sed` actions from `Dockerfile` into `entrypoint.sh`

Reply:
done